### PR TITLE
Adding 2-to-3 example yaml and readme

### PR DESCRIPTION
**Pull request by emchap**
Fixes sourcegraph/sourcegraph#19994.

### CODE REVIEW

**Comment by emchap** [comment]
@malomarrec @LawnGnome I don't know why there's a changed binary showing up here, so if you could confirm if that's an issue (along with whether the changes look okay) I'd be much obliged! :)

File: python-refactor/python-refactor.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+on:
+  - repositoriesMatchingQuery: lang:python print ":[string]" -repo:python-language-server -repo:ctags fork:yes patternType:structural 
+
+# In each repository
+steps:
+  - run: |
+      pip install 2to3 
+      2to3 -w .
+    container: python:3.7.5-slim
```
Comment (LawnGnome) [inline python-refactor/python-refactor.yaml:14]: We may as well track the current stable version, at least for now:

```suggestion
    container: python:3-slim
```

**Review by LawnGnome — APPROVED** [review]
This looks good to me; thanks! I think we could make the step image tag less specific, but I'm not too fussed either way.

In terms of the binary: it looks like you've accidentally committed `.DS_Store`, which is Mac-specific metadata we don't need in the repo. You can remove that with `git rm .DS_Store; git commit; git push`. Alternatively, I can take care of that for you before committing, if that's better for you!

**Comment by emchap** [comment]
@LawnGnome Thank you! Took it out, committed your change to the Python base image. Merging in!

**Comment by malomarrec** [comment]
Thanks @emchap !

**Pull request merged**